cluster/update-storage-objects.sh

```diff
@@ -57,7 +57,7 @@ declare -a resources=(
 )
 
 # Find all the namespaces.
-namespaces=( $("${KUBECTL}" get namespaces -o go-template="{{range.items}}{{.metadata.name}} {{end}}"))
+IFS=" " read -r -a namespaces <<< "$("${KUBECTL}" get namespaces -o go-template="{{range.items}}{{.metadata.name}} {{end}}")"
```

Comment:
Which shellcheck failure are you addressing here?

Reply:
```
In ./cluster/update-storage-objects.sh line 60:
namespaces=( $("${KUBECTL}" get namespaces -o go-template="{{range.items}}{{.metadata.name}} {{end}}"))
             ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
```